desktop/apps/artwork/routes.coffee

```diff
@@ -97,7 +97,7 @@ bootstrap = ->
             res.render 'index', extend data, meData
           .catch next
       else
-        res.render 'index', data
+        res.render 'index', extend data, { isAdmin: true }
```

Comment:
This should actually be `res.locals.sd.CURRENT_USER?.type is 'Admin'` , however locally I'm unable to log into staging for some reason. I was just hardcoding `true` and `false` to check the UI was working as intended for now.

Reply:
So this is going to change before it can be merged?

Reply:
Yes, definitely. I just didn't want to distract myself with figuring out what was up with logging in, and just went with `true`/`false` to make sure the UI will work, once this is fixed.